src/dxvk/dxvk_memory.cpp

```diff
@@ -563,4 +563,59 @@ namespace dxvk {
     }
   }
 
+
+  bool DxvkMemoryAllocator::canShareImage(const VkImageCreateInfo&  createInfo, const DxvkSharedHandleInfo& sharingInfo) const {
```

Comment:
why ist his part of DxvkMemoryAllocator and not DxvkImage? This seems to makes zero sense design-wise.